core/src/main/resources/jenkins/slaves/JnlpSlaveAgentProtocol3/description.jelly

```diff
@@ -1,4 +1,4 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core">
-  ${%Extends the version 2 protocol by adding basic encryption but requires a thread per client}
+  ${%Extends the version 2 protocol by adding basic encryption but requires a thread per client.  This protocol falls back to Java Web Start Agent Protocol/2 (insecure) when it can't create a secure connection.}
```

Comment:
Agreed.  I thought about adding that but started out with a simple description of its behavior.  I'll update the PR based on this feedback.